script.js

```diff
@@ -0,0 +1,112 @@
+const closeBtn = document.getElementById('close');
+const newBookBtn = document.getElementById('new-book-btn');
+const formContainer = document.querySelector('.form-container');
+const backDrop = document.querySelector('.backdrop');
+const form = document.getElementById('form');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const pages = document.getElementById('pages');
+const card = document.getElementById('card');
+const read = document.getElementById('read');
+const myLibrary = [];
```

Comment:
- [ ] Make your library data persistent by saving the books and retrieving them from the[ localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API).